### DISCUSSION TITLE
Translate APP_NAME

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6743] [Q&A] [answered]
How to translate APP_NAME when i switch to another lang, i already added dropdown menu in the CP with en and ar lang but when i switch to arabic the tite still in english. For example if APP_NAME is "Hello" how to make it translated into arabic when i switch to ar to be "أهلا"

**Comment by duncanmcclean** [comment, marked as answer]
Each site can have its own name, configured in the sites config file: https://statamic.dev/multi-site#configuration

You can reference this instead of APP_NAME
